base_report_to_printer/migrations/9.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```

Comment:
jejej, sorry, I've just see the other comment for the same on RMA repository. Done